invadedhouse/models/player.py

```diff
@@ -10,7 +12,7 @@ def __init__(self):
 
     def set_name(self, name: str):
         self.name = name
-        
```

Comment:
Hmm?

Reply:
Same here, just got rid of an extra line